static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -0,0 +1,243 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+import moment from 'moment';
+
+import {getArbitraryRelativePeriod} from 'sentry/components/organizations/timeRangeSelector/utils';
+import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
+import {IconFire} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+import theme from 'sentry/utils/theme';
+import {MetricsTable} from 'sentry/views/performance/trends/changeExplorerUtils/metricsTable';
+import {Chart} from 'sentry/views/performance/trends/chart';
+import {
+  NormalizedTrendsTransaction,
+  TrendChangeType,
+  TrendParameter,
+  TrendsStats,
+  TrendView,
+} from 'sentry/views/performance/trends/types';
+import DetailPanel from 'sentry/views/starfish/components/detailPanel';
+
+type PerformanceChangeExplorerProps = {
+  collapsed: boolean;
+  isLoading: boolean;
+  location: Location;
+  onClose: () => void;
+  organization: Organization;
+  projects: Project[];
+  statsData: TrendsStats;
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+  trendFunction: string;
+  trendParameter: TrendParameter;
+  trendView: TrendView;
+};
+
+type ExplorerBodyProps = {
+  isLoading: boolean;
+  location: Location;
+  organization: Organization;
+  projects: Project[];
+  statsData: TrendsStats;
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+  trendFunction: string;
+  trendParameter: TrendParameter;
+  trendView: TrendView;
+};
+
+type HeaderProps = {
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+};
+
+export function PerformanceChangeExplorer({
+  collapsed,
+  transaction,
+  onClose,
+  trendChangeType,
+  trendFunction,
+  trendView,
+  statsData,
+  isLoading,
+  organization,
+  projects,
+  trendParameter,
+  location,
+}: PerformanceChangeExplorerProps) {
+  return (
+    <DetailPanel detailKey={!collapsed ? transaction.transaction : ''} onClose={onClose}>
+      {!collapsed && (
```

Comment:
I think we can remove this because the collapsed is being handled by the panel, so if it's not showing this stuff doesn't seem to run